clang/test/Driver/sycl-nvptx-link.cpp

```diff
@@ -0,0 +1,19 @@
+// REQUIRES: nvptx-registered-target
+
+// RUN: %clang -### -nocudalib -fsycl -fsycl-targets=nvptx64-nvidia-cuda %s 2>&1 | FileCheck %s
```

Comment:
```suggestion
// RUN: %clang -### -nocudalib -fsycl -fsycl-targets=nvptx64-nvidia-cuda --sysroot=%S/Inputs/SYCL %s 2>&1 | FileCheck %s
```
Adding the internal sysroot should allow for dropping the `REQUIRES: nvptx-registered-target` dependency.

Reply:
Oh I'll give that a go, thanks